app/src/test/java/edu/grinnell/lyles_beers/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package edu.grinnell.lyles_beers;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() throws Exception {
+        assertEquals(4, 2 + 2);
```

Comment:
This is a great example of what a unit test is, but it's not actually useful for us, so delete it.